### DISCUSSION TITLE
Willkommen zu den Daikin-Rotex-HPSU-CAN-Diskussionen!

### DISCUSSION THREAD

**Discussion by Trunks1982** [Trunks1982/Daikin-Rotex-HPSU-CAN#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MichaelGer1410** [comment]


**Reply by Trunks1982** [reply]
Hi du musst ESPHome für Iobroker installieren un dann nur die Ip eingeben im Adapter. Oder ich meine er findet den dann auch automatisch.

**Reply by MichaelGer1410** [reply]
Besten Dank! Adapter ist installiert, die teile aind aus China noch unterwegs dauert noch 6 Tage bis ich weitermachen kann :)
